flexmeasures/utils/config_utils.py

```diff
@@ -64,19 +64,35 @@ def check_app_env(env: str | None):
         "production",
     ):
         print(
-            f'Flask(flexmeasures) environment needs to be either "documentation", "development", "testing", "staging" or "production". It currently is "{env}".'
+            f'Flexmeasures environment needs to be either "documentation", "development", "testing", "staging" or "production". It currently is "{env}".'
         )
         sys.exit(2)
 
 
 def read_config(app: Flask, custom_path_to_config: str | None):
     """Read configuration from various expected sources, complain if not setup correctly."""
 
-    check_app_env(app.env)
+    flexmeasures_env = DefaultConfig.FLEXMEASURES_ENV_DEFAULT
+    if app.testing:
+        print("A")
```

Comment:
Please remove `print` statements here :)

Reply:
removed the print statements